gems/alchemy_cms/CVE-2018-18307.yml

```diff
@@ -0,0 +1,22 @@
+---
+gem: alchemy_cms
+cve: 2018-18307
+ghsa: 7mj4-2984-955f
+url: https://nvd.nist.gov/vuln/detail/CVE-2018-18307
+title: AlchemyCMS is vulnerable to stored XSS via the /admin/pictures image field
+date: 2022-05-14
+description: |
+  A stored XSS vulnerability has been discovered in version 4.1.0 of AlchemyCMS
+  via the /admin/pictures image filename field.
+cvss_v3: 5.9
+unaffected_versions:
+- "< 4.1.0"
+notes: Never patched
```

Comment:
This CVE has been disputed[^1]. It still shows up in every bundle audit. How can I get rid of this?

[^1]: https://nvd.nist.gov/vuln/detail/CVE-2018-18307#match-14738212

Reply:
Created issue #900 to track this. Once the CVE is marked as Rejected or otherwise removed from NVD we can safely remove it here as well.